src/Results.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+import './Results.css'
+
+const Results = ( { character, findImage, clearState, setShowCharacter} ) => {
+  useEffect(() => {
```

Comment:
does this give you an error: React Hook useEffect has a missing dependency: ''. Either include it or remove the dependency array  react-hooks/exhaustive-deps

Mine does when I am using useEffect similarly to this. Just wondering if yours isn't you might be able to help me.